samplekotlin/src/main/AndroidManifest.xml

```diff
@@ -20,9 +20,17 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".ChannelActivity" />
-        <activity android:name=".ChannelActivity2" />
-        <activity android:name=".ChannelActivity3" />
-        <activity android:name=".ChannelActivity4" />
+        <activity android:name=".ChannelActivity"
+            android:theme="@style/AppTheme.NoActionBar"
+            />
```

Comment:
This doesn't look spectacular in the tutorial, but it's not worth adding extra steps for

Reply:
What about changing `AppTheme` from `Theme.AppCompat.Light.DarkActionBar` to `Theme.MaterialComponents.DayNight.NoActionBar` I don't see the need of a `ActionBar`, what do you think?